admin/proc-rhdh-deploy-eks-using-operator.adoc

```diff
@@ -0,0 +1,436 @@
+
+$ sed -i "s#quay.io/fedora/postgresql-15:.*#registry.redhat.io/rhel9/postgresql-15:latest#g" rhdh-operator-*.yaml
+----
+--
+
+. Add the image pull secret to the manifest in the Deployment resource as follows:
++
+--
+[source,yaml]
+----
+--- TRUNCATED ---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: rhdh-operator
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/part-of: rhdh-operator
+    control-plane: controller-manager
+  name: rhdh-controller-manager
+  namespace: rhdh-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        control-plane: controller-manager
+    spec:
+      imagePullSecrets:
+        - name: rhdh-pull-secret
+--- TRUNCATED ---
+----
+--
+
+. Apply the manifest to deploy the operator using the following command:
++
+--
+[source]
+----
+$ kubectl apply -f rhdh-operator-VERSION.yaml
+----
+--
+
+. Run the following command to verify that the Operator is running:
++
+--
+[source]
+----
+$ kubectl -n rhdh-operator get pods -w
+----
+--
+
+== Installing the Developer Hub instance in EKS
+
+Once the Operator is installed and running, you can create a {product-short} instance in EKS.
+
+.Prerequisites
+
+* You have an EKS cluster with AWS Application Load Balancer (ALB) add-on installed. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html[Application load balancing on Amazon EKS] and https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html[Installing the AWS Load Balancer Controller add-on].
+* You have configured a domain name for your {product-short} instance. The domain name can be a hosted zone entry in Route 53 or managed outside of AWS. For more information, see https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/dns-configuring.html[Configuring Amazon Route 53 as your DNS service] documentation.
+* You have an entry in the AWS Certificate Manager (ACM) for your preferred domain name. Make sure to keep a record of your Certificate ARN.
+* You have subscribed to the registry.redhat.io. For more information, see https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication].
+* You have set the context to the EKS cluster in your current `kubeconfig`. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html[Creating or updating a kubeconfig file for an Amazon EKS cluster].
+* You have installed `kubectl`. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html[Installing or updating kubectl].
+
+.Procedure
+
+. Create custom resource file using the following template:
```

Comment:
This procedure has changed as follows:
1. Create the `app-config-rhdh` ConfigMap (containing the RHDH domain name)
2. Create a `secrets-rhdh` Secret
3. Add image pull secret to the default service account
4. Create a Custom Resource referencing both the ConfigMap and Secret resources
5. Create Ingress (containing the RHDH domain name and Certificate ARN)

See https://docs.google.com/document/d/1MWPIaehSAIH-B_peV4UgHuuER-3ZgSbLSLQxomDRfuE/edit#heading=h.upkk6a802726

Reply:
Done